Tobacco and Scents Shenanigans

Linth-Migla opened the pull request:
:cl: Linth
tweak: There is another higher scent level for viscera, and "haze" and "plume" descriptors for smoking.
tweak: Added an all in one rolling kit with tobacco, paper, and filters to cigarette machines.
tweak: Cigarette paper and filter packs are smaller and can fit inside one another and wallets.
tweak: Tobacco rolling pouches can hold any tiny item.
/:cl:

MuckerMayhem reviewed (dismissed):
See this link for making a proper changelog: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation

Also, test failures:
```code/game/objects/items/weapons/storage/wallets.dm:44:error: /: missing comma ',' or right-paren ')'
code/game/objects/items/weapons/storage/wallets.dm:9:error: list started here
code/game/objects/items/weapons/storage/wallets.dm:45:error: ): expected }
code/game/objects/items/weapons/storage/wallets.dm:2:error: location of top-most unmatched {```

MuckerMayhem commented on code/game/objects/items/weapons/storage/wallets.dm:
```diff
@@ -41,6 +41,7 @@
 		/obj/item/passport,
 		/obj/item/clothing/accessory/pride_pin,
 		/obj/item/clothing/accessory/pronouns
```
```suggestion
		/obj/item/clothing/accessory/pronouns,
```

Linth-Migla replied:
The trouble a comma can cause, fixed!

Linth-Migla committed:
Tobacco and Scent Tweaks

Cigarette Paper packets resized to a ‘Tiny Item’.
Fancy Cigarette Paper packets resize to a ‘Tiny Item’.
Cigarette filters packets resized to a ‘Tiny Item’.
Cigarette Paper packets can_hold cigarettes, rolled cigarettes and cigarette filter boxes.
Rolling tobacco pouches now have 10 inventory slots (up from 8). - Unsure if this works! - It does!
Rolling Tobacco pouches should now be able to hold cigarettes, rolled cigarettes, cigarette filter packets and all cigarette paper packets as well as lighters (maybe not lmao- it should be able to hold tiny items). - They can!
—-----
Adds new ‘HAZE’ descriptor for scents!
Adds new ‘PLUME’ descriptor for scents!
Changes Fine Tobacco scent descriptor to ‘plume’.
Changes Terrible Tobacco scent descriptor to ‘haze’.
Adds new ‘overpowering’ scent intensity for organics and synthetics.
—-----
Changes gibs to ‘overpowering’ scent intensity

Delete baystation12Unaltered.dme
LMAO I should not have added this, 3,181 changes - phew

Fixes Wallet Shenanigans
Wallets can_hold tobacco pouches.

squashy boi


Crewman's First Tobacco

Continues tweaks from 'Tobacco and Scent Tweaks'. Add's new tobacco rolling kit with cigarette papers and filters included.

Linth-Migla commented:
Updated to add new tobacco rolling kit item which should be included in the vendor machines. Sorry if this causes headaches.
